### ISSUE TITLE
[Broken Website] Webmin Control Panel

### ISSUE THREAD

**Issue by LoganDark** [darkreader/darkreader#3246]
**Website**
This is a private control panel, so you won't be able to access the dashboard, but here you go.

https://colean.cc:10000

**How to Reproduce**
visit

**Expected behavior**
dark

**Actual behavior**
not, and it doesn't even let me turn it on

Not even on the login screen, so you should be able to test

**Screenshots**
![DARK READER](https://user-images.githubusercontent.com/4723091/88513794-d1821600-cf9d-11ea-9a3a-9e361f209d68.gif)

- OS: macOS 10.14.6
- Browser: Google Chrome 83.0.4103.97 (Official Build) (64-bit)
- Darkreader Version: 4.9.15

**Comment by Gusted** [comment]
Let me guess your site list, it contains `colean.cc`?

**Comment by LoganDark** [comment]
what's a site list?

**Comment by Gusted** [comment]
![](https://i.imgur.com/Fpd9vbg.png)

**Comment by LoganDark** [comment]
![image](https://user-images.githubusercontent.com/4723091/88535427-b5db3780-cfbe-11ea-9ff1-16096f9019bd.png)
It's in the site list

**Comment by LoganDark** [comment]
wow, removing that instantly fixed it. This is not intuitive, clicking the button should remove it from the site list D:

**Comment by Gusted** [comment]
Yep it's a bug with our URL detection should be fixed by #2517(It's on the backlog due it breaking changes and chaning main components of Dark Reader)

**Comment by LoganDark** [comment]
> It's on the backlog due it breaking changes and chaning main components of Dark Reader

Nobody uses Dark Reader as an API, I say merge it and deal with any broken functionality then.

**Comment by Gusted** [comment]
Eventually. Probably the last big update before V5 is released(not sure no guarentees) But the subject of this PR is silenced in the chat group of staff team.
